Formula/woff2.rb

```diff
@@ -0,0 +1,29 @@
+class Woff2 < Formula
+  desc "Utilities to create and convert Web Open Font File (WOFF) files"
+  homepage "https://github.com/google/woff2"
+
+  url "https://github.com/google/woff2/archive/v1.0.2.tar.gz"
+  sha256 "add272bb09e6384a4833ffca4896350fdb16e0ca22df68c0384773c67a175594"
+
+  bottle do
+    sha256 "01a88969d60bbb128a4a73a6b9b6d4c75f4ff08b7c3df45139fc73b7fc76d694" => :high_sierra
+  end
+
+  depends_on "cmake" => :build
+  depends_on "brotli"
+
+  def install
+    system "cmake", ".", *std_cmake_args
+    system "make", "install"
+    bin.install "woff2_info", "woff2_decompress", "woff2_compress"
+  end
+
+  test do
+    # Fetch a woff2 file from Google Fonts
+    system "curl", "-o", "roboto.woff2", "https://fonts.gstatic.com/s/roboto/v18/KFOmCnqEu92Fr1Mu72xKKTU1Kvnz.woff2"
+
+    system "#{bin}/woff2_info", "roboto.woff2"
```

Comment:
Can we check the output of that command, matching some expected string?

Reply:
Not really. Here's an example of what it returns:

	Processing roboto_2 => roboto_2.woff2
	WOFF2Header
	signature           0x774f4632
	flavor              0x00010000
	length              10648
	numTables           14
	reserved            0
	totalSfntSize       20564
	totalCompressedSize 10562
	majorVersion        1
	minorVersion        0
	metaOffset          0
	metaLength          0
	metaOrigLength      0
	privOffset          0
	privLength          0
	TableDirectory starts at +48
	Entry offset flags tag  origLength txLength
	    0     48  0x1a GDEF        206
	    1     51  0x1b GPOS       2370
	    2     54  0x1c GSUB         54
	    3     56  0x06 OS/2         96
	    4     58  0x00 cmap        908
	    5     61  0x11 gasp         12
	    6     63  0x0a glyf      14728    11798
	    7     68  0x0b loca        474        0
	    8     72  0x01 head         54
	    9     74  0x02 hhea         36
	   10     76  0x03 hmtx        944
	   11     79  0x04 maxp         32
	   12     81  0x05 name        372
	   13     84  0x07 post         32
	TableDirectory ends at +86

If you pass it a non-woff2 file, however, it exits with a code 1. So if the conversion fails, so will the test.

I'm building a more robust set of tests which will be part of the updated PR.